tests/FakeItEasy.Tests.ruleset

```diff
@@ -10,6 +10,7 @@
     <Rule Id="CA1020" Action="None" />
     <Rule Id="CA1021" Action="None" />
     <Rule Id="CA1025" Action="None" />
+    <Rule Id="CA1033" Action="None" />

```

Comment:
This is _CA1033: Interface methods should be callable by child types_. FxCop complains because interface members are implemented explicitly without providing a way for derived classes to call them. While it's probably good advice for public APIs, I don't think we'll ever care about that in test projects.